setup.py

```diff
@@ -29,7 +29,7 @@
 install_requires = [
     'setuptools',
     'requests >= 1.0',
-    'ruamel.yaml >= 0.12.4, < 0.15',
+    'ruamel.yaml >= 0.12.4',
```

Comment:
How about changing that to `< 0.16` in case there's another new version?  (ruamel.yaml seems to be treating the minor version as if it were the major version...)